config/processCcd.py

```diff
@@ -3,10 +3,34 @@
 # Configs to get going.
 config.isr.doDark=False
 config.isr.doBias=False
```

Comment:
Don't we have darks and biases now?  If so, should we turn these on to avoid accidentally running without those corrections?